x-pack/platform/plugins/shared/fleet/server/constants/fleet_es_assets.ts

```diff
@@ -126,6 +128,43 @@ export const STACK_COMPONENT_TEMPLATES = [
   STACK_COMPONENT_TEMPLATE_ECS_MAPPINGS,
 ];
 
+export const OTEL_COMPONENT_TEMPLATE_MAPPINGS = 'otel@mappings';
+export const OTEL_COMPONENT_TEMPLATE_SETTINGS = 'otel@settings';
+
+export const OTEL_COMPONENT_TEMPLATE_METRICS_MAPPINGS = 'metrics-otel@mappings';
+export const OTEL_COMPONENT_TEMPLATE_METRICS_CUSTOM_MAPPINGS = 'metrics-otel@custom';
+export const OTEL_COMPONENT_TEMPLATE_LOGS_MAPPINGS = 'logs-otel@mappings';
+export const OTEL_COMPONENT_TEMPLATE_LOGS_CUSTOM_MAPPINGS = 'logs-otel@custom';
+export const OTEL_COMPONENT_TEMPLATE_TRACES_MAPPINGS = 'traces-otel@mappings';
+export const OTEL_COMPONENT_TEMPLATE_TRACES_CUSTOM_MAPPINGS = 'traces-otel@custom';
+export const OTEL_COMPONENT_ECS_TDSB_MAPPINGS = 'ecs-tsdb@mappings';
```

Comment:
I think we don't need to use this template, we should rather use `ecs@mappings`. `ecs-tsdb` looks like a subset of the other.
```suggestion
```